apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -2225,18 +2291,38 @@ static void libc_stdio_nulloutstream_tc(void)
 */
 static void libc_stdio_rawinstream_tc(void)
 {
-	int fd;
+	int fd, ret;
```

Comment:
Please define new variable at different line if there is no specific reason.

[R12] Each variable shall be declared in the new line.

[Example-Do]
```
void example(void)
{
    int i;
    int index;
```
[Example-Don't]

```
void example(void)
{
    int i, index;
```